workflow/usage.rst

```diff
@@ -450,3 +450,216 @@ The following example shows these functions in action:
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd"
+        >
+
+            <framework:config>
+                <framework:workflow name="blog_publishing" type="workflow">
+                    <framework:metadata>
+                        <framework:title>Blog Publishing Workflow</framework:title>
+                    </framework:metadata>
+                    <!-- ... -->
+
+                    <framework:place name="draft">
+                        <framework:metadata>
+                            <framework:max-num-of-words>500</framework:max-num-of-words>
+                        </framework:metadata>
+                    </framework:place>
+                    <!-- ... -->
+
+                    <framework:transition name="to_review">
+                        <framework:from>draft</framework:from>
+                        <framework:to>review</framework:to>
+                        <framework:metadata>
+                            <framework:priority>0.5</framework:priority>
+                        </framework:metadata>
+                    </framework:transition>
+                    <!-- ... -->
+                </framework:workflow>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/workflow.php
+
+        $container->loadFromExtension('framework', array(
+            // ...
+            'workflows' => array(
+                'blog_publishing' => array(
+                    'metadata' => array(
+                        'title' => 'Blog Publishing Workflow',
+                    ),
+                    // ...
+                    'places' => array(
+                        'draft' => array(
+                            'metadata' => array(
+                                'max_num_of_words' => 500,
+                            ),
+                        ),
+                        // ...
+                    ),
+                    'transitions' => array(
+                        'to_review' => array(
+                            'from' => 'draft',
+                            'to' => 'review',
+                            'metadata' => array(
+                                'priority' => 0.5,
+                            ),
+                         ),
+                     ),
+                 ),
+             ),
+         ));
+
+Then, you can access this metadata in your PHP code as follows:
+
+In your Controller::
+
+    public function myControllerAction(Registry $registry, Article $article)
+    {
+        $workflow = $registry->get($article);
+
+        // Or, if you don't inject the Workflow Registry, fetch from the Container:
+        $workflow = $this->get('workflow.article');
+
+        $workflow
+            ->getMetadataStore()
```

Comment:
Code style to discuss, but I think this should be on the line above since the workflow is not being fluent here:
```php
$workflow->getMetadataStore()
    ->getWorkflowMetadata()['title'] ?? false
;
```

Reply:
I'll leave this change until the code style discussion has been had!